charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 3.74.2
+
+* Mount /usr/lib/sysimage/rpm when using host SBOM feature.
```

Comment:
Could we make the release note a little bit more explicit (explaining this is necessary for SBOM feature for hosts running Amazon Linux distros) ?